### ISSUE TITLE
Days to Target function IndentationError

### ISSUE THREAD

**Issue by lucyturn3r** [uccser/codewof#103]
The code below was submitted as an answer to the `Days to Target` function question. 
```
def days_to_target(starting_population, target_population):
    if starting_population - target_population >= 0 :
        return 0
    else:
        keep_going = True
        day_count = 0
        current_population  = starting_population
        
        while keep_going:
            current_population *= 2
            day_count += 1
            if current_population >= target_population:
               	keep_going = False
                return day_count
```
When the code is run locally (I ran it in Wing), the function runs without an error being thrown and all of the test cases pass, but when it is run in codeWOF, I get the following error:
```
IndentationError: unindent does not match any outer indentation level on line 14
                return day_count

                ^
```

I have not been able to figure out why the behaviour is different when the same function is run in these two different environments.

**Comment by lucyturn3r** [comment]
On diffchecker.com, there is an extra (invisible) tab-width character in the code from above which is absent in the code below (which was later submitted, and passed the tests).
```
def days_to_target(starting_population, target_population):
    if starting_population - target_population >= 0 :
        return 0
    else:
        keep_going = True
        day_count = 0
        current_population  = starting_population
        
        while keep_going:
            current_population *= 2
            day_count += 1
            if current_population >= target_population:
                keep_going = False
                return day_count
```
In this image, the code on the left is the code that gave the error, and the code on the right is the code that passed.

![image](https://user-images.githubusercontent.com/27177303/67172261-27c29700-f417-11e9-8723-4a4b97633037.png)

**Comment by lucyturn3r** [comment]
When I convert the original code from unicode to UTF-16, the character shows up as `\u0009`, which is a tab. I don't understand why this tab character is invisible in both the codeWOF editor, and in Wing.

**Comment by courtneycb** [comment]
I'm guessing it doesn't like the mix of spaces and tabs which is causing the indentation error. If I run the code with either all spaces or all tabs it is fine. But as soon as I mix the two it throws an indentation error.

Hopefully there is a setting in CodeMirror to convert tabs to spaces or something similar.

**Comment by courtneycb** [comment]
Overriding the tab character to input spaces works, but not if the code is copy-pasted in.

We tried parsing the users code and replacing tabs with spaces after the code was submitted but because the tab character can have a variable length we couldn't find an appropriate number of spaces to replace it with.

We then tried replacing spaces with tabs but this resulted in an indentation level of 8 spaces (the default tab size).

We are unsure why it works with this particular code in Wing. Python should give the below error if you use tabs with spaces:
![image](https://user-images.githubusercontent.com/16066822/67245849-79bbf900-f4b9-11e9-80b2-e6a898c15250.png)

Maybe we should add a warning if the code detects tabs?
